### PR TITLE
Add Blog post on Bede GH200 benchmarking

### PR DESCRIPTION
**Pull request by ptheywood**
Adds a blog post on GH200 GPUs in Bede, with interconnect bandwidth plot and pytorch benchmarking data / results. 

This is intended to be used for advertising the GH200 nodes to sheffield users, via email(s) once published. 

Targetted publication date Monday 2024-05-06

### CODE REVIEW

File: _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "Bede Tier 2 HPC: Nvidia Grace-Hopper Superchip Pilot"
+author: Peter Heywood
+slug: 2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip
+date: 2024-05-06 12:00:00 UTC
```
Comment (Robadob) [inline _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md]: Monday is a Bank Holiday

File: _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+<!--more-->
+
+The following figure shows the theoretical peak bandwidth for the range of GPU interconnect technologies used in a range of GPUs.
+
+![Figure 1: GPU host-device interconnect theoretical peak bandwidth](/assets/images/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip/gpu-interconnects-tuos.png)
+Source: [github.com/ptheywood/gpu-interconnect-plots][ptheywood/gpu-interconnect-plots]
+
+### PyTorch LLM Fine-tuning Benchmark
+
+To illustrate the performance of the GH200 GPUs for machine learning workloads, a benchmark of LLM fine-tuning previously used by [Farhad Allian][farhad-github] to [investigate the performance of NVIDIA L40 GPUs for machine learning][l40-pytorch-benchmark-blog] benchmarked on the [GH200 GPUs in Bede][bede-gh200-pilot-docs].
```
Comment (Robadob) [inline _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md]: Worth clarifying in TEXT he's RIT shef?

File: _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+Source files, instructions, job submission scripts and the generated results and figures can be found in the [RSE-Sheffield/pytorch-transformers-wikitext2-benchmark GitHub repository][RSE-Sheffield/pytorch-transformers-wikitext2-benchmark].
+
+#### FP32 Results
+
+The following figures and tables show the benchmark data for **FP32** training and inferencing on across a range of GPUs.
+
+This includes the runtime training and inferencing phases in seconds (lower is better),
+and the samples processing rate in samples per second (higher is better).
+
+As you might expect, newer generations of GPU offer reduced application runtimes and increased performance compared to previous generations, with the GH200 outperforming the V100 SXM2 GPUs in Bessemer, the A100 SXM4 GPUs in Stanage and the H100 PCIe GPUs in stanage.
```
Comment (Robadob) [inline _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md]:
```suggestion
As you might expect, newer generations of GPU offer reduced application runtimes and increased performance compared to previous generations, with the GH200 outperforming the V100 SXM2 GPUs in Bessemer, the A100 SXM4 GPUs in Stanage and the H100 PCIe GPUs in Stanage.
```
Or just reword so Stanage is only mentioned once.

**Review by Robadob — APPROVED** [review]
Left a few notes, only looked at the Markdown

File: _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md (hunk 1 above)
Comment (ptheywood) [reply _posts/2024-05-06-bede-tier-2-hpc-nvidia-grace-hopper-superchip.md]: True, I'd forgotten that last week apprently. I'm off ill today so will make the suggested changes and update dates for publication hopefully tomorrow.

**Pull request merged**